### DISCUSSION TITLE
When I input Chinese in the search box, the input box is unrecognized

### DISCUSSION THREAD

**Discussion by tuntuntutu** [requarks/wiki#3683] [Error / Bug Report]
**Describe the bug**
When I input Chinese in the search box, the input box is unrecognized and becomes null directly after losing focus. But when I paste Chinese, I don't have this problem

**To Reproduce**
1. type Chinese words in search box，for example ’你好‘
2. lose focus
3. search box value will be null

**Expected behavior**
search box value should be ’你好‘

**Host Info (please complete the following information):**
 - OS: Linux - CentOS (core) 7.6.1810 x64
 - Wiki.js version: 2.3.81
 - Database engine: 5.7.28

**Comment by liudonghua123** [comment]
It seems related to https://github.com/requarks/wiki/issues/1404. Need to config the Postgres. See also https://zhuanlan.zhihu.com/p/335359081.
